website/docs/reference/project-configs/clean-targets.md

```diff
@@ -19,10 +19,11 @@ Optionally specify a custom list of directories to be removed by the `dbt clean`
 If this configuration is not included in your `dbt_project.yml` file, the `clean` command will remove files in your [target-path](/reference/project-configs/target-path).
 
 ## Examples
+
 ### Remove packages and compiled files as part of `dbt clean`
-:::info
+
 This is our preferred configuration, but is not the default.
```

Comment:
Maybe we can even remove this line and add "preferred" in the example title?

Reply:
Works for me!